libs/island-ui/core/src/lib/Breadcrumbs/Breadcrumbs.treat.ts

```diff
@@ -1,5 +1,62 @@
-import { style } from 'treat'
+import { style, styleMap } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
 
 export const divider = style({
   margin: '0 10px',
 })
+
+export const wrapper = style({
+  display: 'inline-flex',
+  alignItems: 'center',
+})
```

Comment:
Should be handled with a Box component.
Most likely same with the divider style.